Sources/Node/Convertible/Date+Convertible.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+extension Date: NodeConvertible {
+    /**
+     Turn the Date into a node
+     
+     - throws: if convertible can not create a Node
+     - returns: a node if possible
+     */
+    public func makeNode(context: Context) throws -> Node {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
```

Comment:
I can only suggest not using ints. one day will come where you are debugging in the db. And there is nothing worse than having unix time stamps there! This is awesome! Only downside is the format is hardcoded for MySQL. Mongo, Elastic have different default formats